src/components/breadcrumbs/_breadcrumbs.scss

```diff
@@ -89,18 +83,35 @@
 }
 
 .ouiBreadcrumbWrapper {
-  background-color: $ouiBreadcrumbGrayBackground;
-  transform: skewX(-20deg);
-  border-radius: $ouiSizeXS;
+  position: relative;
+  z-index: 0;
   padding: $ouiSizeXS - 2.5 $ouiSizeL - $ouiSizeXS;
+  padding-right: $ouiSizeL - $ouiSizeXS + $ouiBreadcrumbSpacing / 2;
+
+  &::before {
+    content: '';
+    position: absolute;
+    top: 0;
+    bottom: 0;
+    left: $ouiBreadcrumbSpacing / 2;
+    right: $ouiBreadcrumbSpacing / 2;
+    z-index: -1;
+
+    transform: skewX(-20deg);
+    border-radius: $ouiSizeXS;
+  }
 
-  &:not(.ouiBreadcrumbWrapper--first) {
-    margin-bottom: $ouiSizeXS; /* 1 */
+  &:not(.ouiBreadcrumbWrapper--last)::before {
+    background-color: $ouiBreadcrumbGrayBackground;
   }
 
-  &:not(.ouiBreadcrumbWrapper--last),
-  &.ouiBreadcrumbWrapper--first.ouiBreadcrumbWrapper--last {
-    margin-right: $ouiBreadcrumbSpacing;
+  &.ouiBreadcrumbWrapper--last::before {
+    background-color: $ouiBreadcrumbBlueBackground;
+  }
+
+  &:not(.ouiBreadcrumbWrapper--first) {
+    margin-bottom: $ouiSizeXS; /* 1 */
```

Comment:
Did we at some point lose the comment that explains this? It's not obvious why all the wrappers would have a `margin-bottom` except the first one.

Reply:
It's still there, just at the very top of the file:

https://github.com/opensearch-project/oui/blob/1a52dc3dd730be8da4b6eff194ed7771d99e3dbd/src/components/breadcrumbs/_breadcrumbs.scss#L12-L15

Reply:
Also, the first one doesn't have it because the margin is added to the first one here:

https://github.com/opensearch-project/oui/blob/1a52dc3dd730be8da4b6eff194ed7771d99e3dbd/src/components/breadcrumbs/_breadcrumbs.scss#L122

Reply:
Oh, so is it possible for breadcrumbs to wrap? I don't think I've tested that behavior before, but it makes sense.

Reply:
Technically, yes, if you don't use truncation. The "Truncation on a single item" example on the docsite shows this